backend/src/test/java/com/google/sps/UserCrudTest.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.ArrayList;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class UserCrudTest {
+  private static final LocalServiceTestHelper HELPER =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private static final String EMAIL = "testEMAIL@gmail.com";
+  private static final String EMAIL_2 = "testEMAIL_2@gmail.com";
+  private static final Long TRIP_ID = 111222333L;
+  private static final ArrayList<String> EMPTY_TRIP_IDS = new ArrayList<String>();
+  private static final ArrayList<String> SINGLE_TRIP_ID = new ArrayList<String>() {
+    { add(Long.toString(TRIP_ID)); }
+  };
```

Comment:
nit: Import `java.util.Arrays` and you can do:
```suggestion
  private static final List<String> EMPTY_TRIP_IDS = new ArrayList<String>();
  private static final List<String> SINGLE_TRIP_ID = Arrays.asList({Long.toString(TRIP_ID)});
```

Reply:
done